workspace/TS100/configuration.h

```diff
@@ -131,7 +131,7 @@
 
 #ifdef MODEL_TS100
 const int32_t tipMass = 45; // X10 watts to raise 1 deg C in 1 second
-const uint8_t tipResistance = 85; //x10 ohms, 8.5 typical for ts100, 4.5 typical for ts80
+const uint8_t tipResistance = 75; //x10 ohms, 7.5 typical for ts100 tips
```

Comment:
I find this interesting, a long time ago now I did four terminal measurements across all of the tips I had then and they were all ~8.5 ohms. I wonder if thats dropped over time as newer ones are made. What did you use to measure this?

Reply:
As I described in #693, I was using a UNI-T T60A DMM, making sure it gave me exactly 0.0 Ohm when shorting the probes.

Reply:
I recently picked up a whole new set of tips, and sure enough I have a spread of 7.3 to 9 ohms across them all.
Happy to drop this to 7.5 :)

Reply:
Thank you for checking your set. I think it makes sense to have this close to the lower bound as it directly affects power limiting accuracy.

Reply:
Absolutely agree, and there is always going to be some tolerance band anyway, so why not go with the most useful end :)